src/main/java/dataLoader/jsonDataLoader.java

```diff
@@ -0,0 +1,40 @@
+package dataLoader;
+
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Simple Class that parses data set from a json file
+ * without knowing the files format
+ * & prints out the key value pairs.
+ */
+
+
+
+public class jsonDataLoader {
```

Comment:
Classes in Java by convention start with upper case and then follow CamelCase. So please rename to ```JsonDataLoader```